docs/methods.rst

```diff
@@ -1206,6 +1206,43 @@ the values found in the \*.sXXX.opt.h5 file. Be careful to keep the pair
 of optimized CI coefficients and Jastrow coefficients together to avoid
 inconsistencies.
 
+Parameter gradients
+~~~~~~~~~~~~~~~~~~~
+The gradients of the energy with respect to the variational parameters can be checked and optionally written to a file.
+The check compares the analytic derivatives with a finite difference approximation.
+These are activated by giving a ``gradient_test`` method in and ``optimize`` block, as follows:
+
+::
+
+     <qmc method="linear" move="pbyp">
+      <optimize method="gradient_test">
+      </optimize>
+      ... rest of optimizer input ...
```

Comment:
`rest of optimizer input`->`VMC related input`?
With gradient_test, I doubt any other optimizer parameters are still needed.

Reply:
This is probably true, but I expect the common case is starting from an existing optimizer input block.